.env.default

```diff
@@ -22,4 +22,11 @@ SUBGRAPH_COMPONENT_RETRIES=1
 
 DISABLE_THIRD_PARTY_PROVIDERS_RESOLVER_SERVICE_USAGE=false
 
+L1_LAND_REGISTRY_CONTRACT_ADDRESS=0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d
+L2_LAND_REGISTRY_CONTRACT_ADDRESS=0x959e104e1a4db6317fa58f8295f586e1a978c297
```

Comment:
why? we don't have lands o estates in l2

Reply:
I wanted to select between the mainnet and the test-network addresses, but mistakenly named as that. I'll get it fix, great catch!

Reply:
I'm retrieving now the contracts from a catalysts dependency using the ETH_NETWORK defined in the catalysts.